spec/forms/concerns/form_attributes_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rails_helper'
+
+class ExampleModel
+  # This was copied from OutgoingEmail to have a real example, but not literally
+  # tie to an existing model (which may change)
+  def self.column_defaults
+    {
+      "id" => nil,
+      "body" => "some body text",
+      "client_id" => nil,
+      "created_at" => nil,
+      "mailgun_status" => "sending",
+      "message_id" => nil,
+      "sent_at" => nil,
+      "subject" => nil,
+      "to" => nil,
+      "updated_at" => nil,
+      "user_id" => nil
+    }
+  end
```

Comment:
just verifying that this `column_defaults` exist on a model by default -- using the database definitions of what the default is for these fields, is that correct?

Reply:
yes, i didn't know about this either! https://apidock.com/rails/ActiveRecord/Base/column_defaults/class